build.gradle

```diff
@@ -257,6 +240,23 @@ project('kcbq-api') {
 }
 
 project('kcbq-confluent') {
+    apply plugin: 'distribution'
+
+    configurations.all {
+        resolutionStrategy {
+            force 'org.apache.kafka:connect-api:1.1.1', 'org.apache.kafka:kafka-clients:1.1.1'
+        }
+    }
+
+    distributions {
+        main {
+            baseName = 'kafka-connect-bigquery'
```

Comment:
nit: name kcbq-confluent .. theoretically, if we had another schema reg implementation (e.g. proto), we could then have a kcbq-proto dist package in another project as well. I prefer naming the same as the submodule.